tests/extractor_test.py

```diff
@@ -0,0 +1,91 @@
+from pytest import raises
+from mock import patch
+from demisto_sdk.common.configuration import Configuration
+from demisto_sdk.common.constants import DEFAULT_IMAGE_BASE64
+import os
+import base64
+
+
+def test_get_yml_type():
+    from demisto_sdk.yaml_tools.extractor import Extractor
+    configuration = Configuration()
+    with patch.object(Extractor, '__init__', lambda a, b, c, d, e, f, g: None):
+        extractor = Extractor('', '', False, False, '', configuration)
```

Comment:
Can we create this instance once for all the tests?

Reply:
refactored, please take a look

Reply:
@guyfreund I meant all the instance creation in the unifier and extractor, can it be done one time only?